doc/nrf/known_issues.rst

```diff
@@ -679,6 +679,16 @@ Closing sockets
 Multiprotocol Service Layer (MPSL)
 ==================================
 
+.. rst-class:: v1-4-99-dev1 v1-4-2 v1-4-1
+
+DRGN-15223: Do not use `CONFIG_SYSTEM_CLOCK_NO_WAIT` with nrf53.
```

Comment:
```suggestion
DRGN-15223: `CONFIG_SYSTEM_CLOCK_NO_WAIT` is not supported for nRF5340
   Using :option:`CONFIG_SYSTEM_CLOCK_NO_WAIT` with nRF5340 devices might not work as expected.

```

Reply:
No need for a workaround in this case - if it is not supported, of course you should not use it. ;)

Reply:
Thank you. Addressed.